docs/mitigations.md

```diff
@@ -0,0 +1,15 @@
+General 
```

Comment:
```suggestion
# Mitigations
Temporary mitigations to prevent infections and stop the malware from running. These are not a permenant solution.

## General 
```